packages/components/CONTRIBUTING.md

```diff
@@ -20,6 +20,60 @@ To run the unit tests for the components, run:
 ```bash
 pnpm --filter design-system-components test
 ```
+## Writing Components
+
```

Comment:
Include a section about the general styleguide of stencil https://stenciljs.com/docs/style-guide and that ~~we're~~ we should extend it.